Source/Scene/ModelAnimation.js

```diff
@@ -230,4 +240,24 @@ Object.defineProperties(ModelAnimation.prototype, {
     },
   },
 });
+/**
+ * A function used to compute the local animation time for a ModelAnimation.
```

Comment:
Clarify that if this function is defined by the user, the animation will use this instead of the scene time to evaluate its animation.

Reply:
But this is the type definition for the callback. I *do* explicitly clarify that at the point that the animationTime field is defined (at line 42 above).